From af67f0d12aff6f6000212a88a5e4256fb1216655 Mon Sep 17 00:00:00 2001
From: Delphine CC Chiu <Delphine_CC_Chiu@wiwynn.com>
Date: Tue, 26 Sep 2023 14:24:08 +0800
Subject: [PATCH 02/18] PSUSensor: add adm1281 support

add device adm1281 pmbus reading support

Tested:
Get adm1281 reading from PSUSensor by dbus introspect

Change-Id: Ifca216a4dc7a8457bbaa0894e228bd22b9b418bb
Signed-off-by: Delphine CC Chiu <Delphine_CC_Chiu@wiwynn.com>
---
 src/psu/PSUSensorMain.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/psu/PSUSensorMain.cpp b/src/psu/PSUSensorMain.cpp
index f0f6f05..133e45f 100644
--- a/src/psu/PSUSensorMain.cpp
+++ b/src/psu/PSUSensorMain.cpp
@@ -69,6 +69,7 @@ static const I2CDeviceTypeMap sensorTypes{
     {"ADM1272", I2CDeviceType{"adm1272", true}},
     {"ADM1275", I2CDeviceType{"adm1275", true}},
     {"ADM1278", I2CDeviceType{"adm1278", true}},
+    {"ADM1281", I2CDeviceType{"adm1281", true}},
     {"ADM1293", I2CDeviceType{"adm1293", true}},
     {"ADS1015", I2CDeviceType{"ads1015", true}},
     {"ADS7830", I2CDeviceType{"ads7830", true}},
-- 
2.47.3

